sdks/python/apache_beam/dataframe/transforms.py

```diff
@@ -302,6 +302,7 @@ def __repr__(self, indent=0):
                 self.outputs))
 
     # First define some helper functions.
+    @_memoize
```

Comment:
What is the lifetime for cached entries?

Reply:
The lifetime is that of the call of the enclosing function. (These are closures.)

Reply:
I see, thanks, looks like this is used only during graph creation.